requirements-dev.txt

```diff
@@ -8,6 +8,7 @@ charset-normalizer==3.2.0
 click==8.1.6
 colorama==0.4.6
 coverage==7.3.1
+craft-application@git+https://github.com/canonical/craft-application.git
```

Comment:
Looks like rockcraft is doing this too, so I am fine with this getting merged into a feature branch.

Reply:
This pulse is probably an appropriate time for a release of craft-application too, which will let us merge this feature branch.